tests/fixtures/long.md

```diff
@@ -17,16 +17,33 @@ function foo() {
 }
 ```
 
+<!-- eslint-env node -->
+<!-- eslint-disable eol-last, quotes -->
+
 ```node
 console.log(process.version);
 ```
 
 How about some JSX?
 
+<!--
+    eslint quotes: [
+        "error",
+        "single"
+    ]
+-->
+<!--eslint-disable no-console-->
+
 ```jsx
 console.log("Error!");
 ```
 
     I may be a code block, but don't lint me!
 
+<!-- eslint-disable -->
```

Comment:
i guess if `eslint-disable` only affects the next code block - as opposed to the entire file it appears in (like in JS files), then there's no need.